### PR TITLE
Prepare to register

### PR DESCRIPTION
**Pull request by darsnack**
Does some clean up and adds docs.

### CODE REVIEW

**Comment by darsnack** [comment]
I assume we can drop the GPUArrays testing and just use build kite?

**Comment by mcabbott** [comment]
My hope was that we could leave GPUArrays testing live, so that testing on a machine without a GPU will still run the GPU tests -- rather than waiting for CI to tell you what's wrong.

File: test/gpu.jl [hunk 1]
```diff
@@ -6,7 +6,7 @@
   cx = cu(x)
   @test cx isa CuArray
 
-  @test_broken onecold(cu([1.0, 2.0, 3.0])) == 3  # scalar indexing error?
+  @test onecold(cu([1.0, 2.0, 3.0])) == 3
```
Comment (mcabbott) [inline test/gpu.jl:9]:
```suggestion
  @test_skip onecold(cu([1.0, 2.0, 3.0])) == 3  # passes with CuArray with Julia 1.6, but fails with JLArray
```

**Comment by darsnack** [comment]
I recall seeing the build kite error on Flux, and I think @ToucheSir might have fixed it?

**Comment by ToucheSir** [comment]
Yup, you need to follow the instructions in https://github.com/juliagpu/buildkite#using-secrets to add the encrypted codecov token to the buildkite config.

**Review by ToucheSir — APPROVED** [review]

**Comment by ToucheSir** [comment]
Who wants to do the honours?

**Pull request merged**